src/App.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Switch, Route, Redirect } from "react-router-dom";
+import PrivateRoute from "./components/PrivateRoute/PrivateRoute";
+import routes from "./routes/routes";
```

Comment:
I would personally prefer to have a index.ts that export the component instead of having all /{componetfolder}/{component name}

Example: "import routes from "./routes";"
where in routes you will have a index.ts
that has export {Route} from './routes.tsx'